apps/network-alerts/js/attention/attention.js

```diff
@@ -42,7 +42,10 @@ function sendNotification() {
         navigator.mozL10n.get(title), {
           body: body,
           tag: '' + Date.now(), // needs to be unique
-          icon: NotificationHelper.getIconURI(app) + '?titleID=' + title
+          icon: NotificationHelper.getIconURI(app),
+          data: {
+            "title": title
```

Comment:
__nit:__ we don't use quotes for object properties unless it's JSON file, so I'd rather have ```data: { title }``` - notice new ES6 shorthand syntax [1] ;)

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer